backend/amd64/CSE.ml

```diff
@@ -20,37 +20,32 @@ open! Int_replace_polymorphic_compare
 
```

Comment:
Not really about this pull request, but
`[@@@ocaml.warning ...]` above should
probably be in line with the other files.